packages/admin/src/table/filterbar/FilterBarActiveFilterBadge.tsx

```diff
@@ -0,0 +1,60 @@
+import { createStyles, Theme, Typography, WithStyles, withStyles } from "@material-ui/core";
+import * as React from "react";
+
+export type CometAdminFilterBarActiveFilterBadgeClassKeys = "hasValueCount";
+
+export const dirtyFieldsCount = (values: Record<string, any>) => {
+    return Object.values(values).reduce((acc, val) => {
+        if (val) {
+            if (Array.isArray(val)) {
+                acc += val.length;
+            }
+            acc++;
+        }
+        return acc;
+    }, 0);
+    /*
+    TODO: what is the use case of this?
+    if (fieldState?.initial) {
+        const diff = difference<any, Record<string, any>>(fieldState?.value, fieldState?.initial);
+        return Object.keys(diff).length;
+    }
+    */
```

Comment:
die difference Funktion wurde gebraucht um ein Object zu bekommen welches nur aus dem Unterschied von initial und derzeitigen FormValues besteht um dann die Anzahl an geänderten Formfeldern zählen zu können :) 
Dein Weg ist aber auch sehr gut und sehr intressant :) hast du deinen ansatz auch mit zB mit meinem RangeSlider ausprobiert ? dieser besteht ja aus min/max Value :)

Reply:
nein hab ich nicht probiert.

aber mein gedanke war, dass selbst wenn im initial was drin steht, möchte ich das im badge sehen

Reply:
wenn du das in deinen PR übernehmen willst, kannst du den PR mergen, den commit cherry-picken oder einfach nur hier den diff anschaun und nochmal neu implementieren. wie es für dich besser passt :)